administrator/templates/atum/scss/blocks/_form.scss

```diff
@@ -128,4 +128,39 @@ legend {
   display: block;
   position: absolute;
   bottom: 100%;
+}
```

Comment:
none of this should be necessary - switcher has its own scss

Reply:
this still needs to be resolved

Reply:
The base switcher css is not changed here anymore. Just the legend is adjust by the font size of the template (which is a template thing I think).

Reply:
ok